src/GovUk.Education.ExploreEducationStatistics.Publisher.Model/GovUk.Education.ExploreEducationStatistics.Publisher.Model.csproj

```diff
@@ -8,4 +8,7 @@
       <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="1.0.5" />
     </ItemGroup>
 
+    <ItemGroup>
```

Comment:
I see new Project-to-common dependencies being added in, but I was also expecting to see dependencies coming out (as more projects are now dependent on common).  Are we not yet ready to do this?

Reply:
No, unfortunately, as discussed, a lot of the dependencies are needed but the Model projects should now only be dependent on Common and also the dependency that was causing the biggest problem for me between Content.Model and Data.Model has been removed.

If the queue message classes were moved to either Common or a new library we might be able to remove some of the other dependencies.